scripts/deploy-tokenizedBallot.ts

```diff
@@ -0,0 +1,45 @@
+import 'dotenv/config';
+import {ethers} from 'hardhat';
+import {TokenizedBallot__factory} from '../typechain-types';
+
+export const deployTokenizedBallot = async (): Promise<void> => {
+  const {ALCHEMY_API_KEY} = process.env;
+  const GOERLI_PRIVATE_KEY = process.env.GOERLI_PRIVATE_KEY as string; // getting linter issues if not assigning type.
```

Comment:
Could also do
`if (typeof process.env.GOERLI_PRIVATE_KEY !== "string) return`

Reply:
Is this better practice? For what reason?
Please see changes.
Actually the linter issue wasn't reproduced while I was doing this with:

Reply:
> Is this better practice? For what reason?

Yes, because casting it as a string is telling typescript it's a string, but it may actually be undefined, and you're not handling that case. It would trigger an error further down the stack, and could be harder to debug. This way, you're dealing with the error explicity.

Reply:
Got it, thanks!